### DISCUSSION TITLE
Add more explicit details about acq PC set-up?

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_experiments#276] [General]
These are my (more thorough but still probably incomplete) notes for the steps involved for an acq PC set up. Maybe should be added to the README or somewhere else

- Update mobo bios via flashing via .efi
- Format disks as RAID0 via BIOS
- Install Windows 10 LTSC on non-RAIDED disk
    - Create user: aeon
    - password: Project_&lt;HOSTNAME&gt;
- Update ALL requisite mobo drivers within Windows
- Connect to VLAN
    - In Control Panel -> Network and Internet -> Network and Sharing Center -> Ethernet Status -> Ethernet Properties -> IPV4 Properties
- Turn off Windows sleep and update settings
- Map Ceph
- Install BeyondTrust Jump Client (and optionally Access Console)
- Install prerequisites mentioned in README
    - Install Bonsai system version from bonsai-rx.org (this should probably be added to prerequisites)
- Clone 'aeon\_experiments' and 'aeon\_acquisition' in 'C:/ProjectAeon/'
- Follow repos set up instructions
    - need to add run `python 'setup.ps'`
- Create a data directory on RAIDed drives: D:/Data/ProjectAeon
- Install brainboxes E2S
- Manually set Com ports to the device ports specified within the Bonsai exp workflow

@glopesdev @aspaNeuro

**Comment by aspaNeuro** [comment]
We will now install new AEON Machines and Ill try to create a document for all these components
